core-rust/state-manager/src/store/typed_cf_api.rs

```diff
@@ -738,6 +767,41 @@ impl DbCodec<()> for UnitDbCodec {
     }
 }
 
+/// A [`DbCodec]` capable of representing [`MigrationId]`.
```

Comment:
Out of interest, why did we opt for manual codecs rather than SBOR encoding the id and migration status?

Reply:
Good point. I've created them at the beginning, when I had a very basic understanding of how things done here. Will change to SBOR encoding.

Reply:
Fixed